src/main/scripts/create-ds.sh

```diff
@@ -57,6 +59,19 @@ if [ $dbType == "db2" ]; then
     sed -i "s#\${DB2_DATASOURCE_JNDI_NAME}#${jdbcDSJNDIName}#g" $createDsScript
     sed -i "s/\${DB2_SERVER_NAME}/${db2ServerName}/g" $createDsScript
     sed -i "s/\${PORT_NUMBER}/${db2ServerPortNumber}/g" $createDsScript
+elif [ $dbType == "oracle" ]; then
+    # Download jdbc drivers
+    curl -Lo ${jdbcDriverPath}/ojdbc8.jar https://download.oracle.com/otn-pub/otn_software/jdbc/1916/ojdbc8.jar
+    jdbcDriverClassPath=$(realpath "$jdbcDriverPath"/ojdbc8.jar)
```

Comment:
I'm sorry to do this, but I need you to put this inside a retry loop.

Reply:
Good point. Fixed in [85a7df9](https://github.com/WASdev/azure.websphere-traditional.singleserver/pull/55/commits/85a7df970a16d683b442045a943a757260818945).